melos.yaml

```diff
@@ -3,6 +3,7 @@ repository: https://github.com/flame-engine/gamepads
 
 packages:
   - packages/**
+  - packages/**/example
```

Comment:
The first glob already includes these

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/flame-engine/gamepads/assets/6718144/9e24fb8d-e178-4f9f-be0e-ee0429135dc9)

Reply:
```suggestion
```